location_scrapbook/app/src/main/java/erica/locationscrapbook/DBService.java

```diff
@@ -81,8 +73,6 @@ public void deleteLoc (Marker marker) {
 
             locList.add(locs);
 
-
-
```

Comment:
This might be a personal thing, but I think code looks cleaner without a bunch of empty lines like these